releasenotes/notes/injection-guardrails-bde1d57db91f33d1.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    automatic instrumentation: Introduces generic safeguards for automatic instrumentation ("library injection") in the form of early exit conditions.
```

```suggestion
    SSI: Introduces generic safeguards for automatic instrumentation when using single step install in the form of early exit conditions.
```

To differentiate this feature from what we normally call automatic instrumentation should we say `single step install` ?